README.md.template

```diff
@@ -1,16 +1,16 @@
 _This template includes more information than a typical project requires, both to provide hints on possible things to include, as well as to make the process of filling it largely a matter of deleting information that is not applicable. Specifically; **be sure to remove or replace any notes and comments in italics,** like this one. By convention, pseudo-variables you should replace are typically in ALLCAPS._
 
 
-# [{{PROJECT_TITLE}}](http://{{PROJECT_REPO_URL:github.com/loadsys/app}})
+# [{{PROJECT_TITLE:@TODO}}]({{PROJECT_REPO_URL:https://github.com/loadsys/app}})
 
-{{PROJECT_DESCRIPTION}}
+{{PROJECT_DESCRIPTION:@TODO}}
 
 _Brief app description. Why does it exist? Who uses it?_
 
-* Production URL: {{PROJECT_PRODUCTION_URL}}
-* Staging URL: {{PROJECT_STAGING_URL}}
-* Project Management URL: {{PROJECT_MANAGEMENT_URL}}
-* Loadsys Project Docs: {{PROJECT_DOCUMENT_URL}}
+* Production URL: {{PROJECT_PRODUCTION_URL:@TODO}}
+* Staging URL: {{PROJECT_STAGING_URL:@TODO}}
+* Project Management URL: {{PROJECT_MANAGEMENT_URL:@TODO}}
+* Loadsys Project Docs: {{PROJECT_DOCUMENT_URL:@TODO}}
```

Comment:
@alecho This is a good example of one of the principles I'm trying to explain. An empty "Production URL" value **is not acceptable**. We **should not** allow a developer to skip through that prompt during token replacement without "punishing" them for their laziness by leaving an ugly `@TODO` marker behind for them to clean up by hand.

To let them get away with this is sloppy on **our** part, as the designer of the tool we expect them to use. We gave them an easy way to do it, and they skipped it, leading to more work (with or without the `TODO` markers, that's still true). Our job is to reinforce the easy way though and leaving the markers makes their laziness obvious to everyone.

Most of why I'm arguing so fiercely about a lot of this stuff can be drawn directly from this principle.